src/PersonLink.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from './types';
+
+const FATHER = 'father';
+const MOTHER = 'mother';
+const FEMALE = 'f';
+const NOT_SET_VALUE = '-';
+
+interface ParentLink {
+  person: Person,
+  parentType: string,
+  name: string,
+  peopleList: Person[],
+}
+
+const getParentLink = (
+  args: ParentLink,
```

Comment:
```suggestion
  props: ParentLink,
```
We usually name it `props`